src/drivers/distance_sensor/vl53lxx/vl53lxx.cpp

```diff
@@ -784,32 +642,80 @@ VL53LXX::spadCalculations()
 
 	sensorTuning();
 
-	writeRegister(SYSTEM_INTERRUPT_CONFIG_GPIO_REG, 4); 		// 4: GPIO interrupt on new data
-
+	writeRegister(SYSTEM_INTERRUPT_CONFIG_GPIO_REG, 4);		// 4: GPIO interrupt on new data.
 	readRegister(GPIO_HV_MUX_ACTIVE_HIGH_REG, val);
-	writeRegister(GPIO_HV_MUX_ACTIVE_HIGH_REG, val & ~0x10);  	// active low
 
+	writeRegister(GPIO_HV_MUX_ACTIVE_HIGH_REG, val & ~0x10);	// Active low.
 	writeRegister(SYSTEM_INTERRUPT_CLEAR_REG, 0x01);
-
 	writeRegister(SYSTEM_SEQUENCE_CONFIG_REG, 0xE8);
-
 	writeRegister(SYSTEM_SEQUENCE_CONFIG_REG, 0x01);
+
 	singleRefCalibration(0x40);
 
 	writeRegister(SYSTEM_SEQUENCE_CONFIG_REG, 0x02);
-	singleRefCalibration(0x00);
 
-	writeRegister(SYSTEM_SEQUENCE_CONFIG_REG, 0xE8); 			// restore config
+	singleRefCalibration(0x00);
 
-	return OK;
+	writeRegister(SYSTEM_SEQUENCE_CONFIG_REG, 0xE8);		// Restore config.
 
+	return PX4_OK;
```

Comment:
PX4_OK is not the correct return value when returning bool

Reply:
Thanks @LorenzMeier , you are absolutely correct.  Nice catch!  The method return remains the same value, but it was certainly wrong previously.

For the convenience of other readers: from `px4_defines.h`: 
```
#define OK 0
#define ERROR -1
```

To preserve the return value I have changed `bool` method return types to `int`, this matches convention in this and other driver method return types.  If you would prefer otherwise just let me know!

Thanks again for catching this!!

Rebased on current master and bench tested.